HeatStroke/Graphics/Particles/Emitter.cpp

```diff
@@ -0,0 +1,702 @@
+#include "Emitter.h"
+
+#include "AffectorFade.h"
+#include "AffectorColor.h"
+#include "AffectorScale.h"
+#include "AffectorVelocity.h"
+#include <cassert>
+
+// static member setup
+std::random_device HeatStroke::Emitter::s_Rand;
+std::mt19937 HeatStroke::Emitter::s_RNGesus = std::mt19937(HeatStroke::Emitter::s_Rand());
```

Comment:
Just a temp name.  Would like to create a Random class that abstracts this away (though it will internally use s_RNGesus)